fastlane/lib/fastlane/environment_printer.rb

```diff
@@ -128,6 +129,25 @@ def self.print_loaded_gems
       return env_output
     end
 
+    def self.print_system_locale
+      env_output = "### System Locale\n\n"
+      table = "| Variable | Value |  |\n"
+      table << "|-----|---------|----|\n"
+      ["LANG", "LC_ALL", "LANGUAGE"].each do |e|
+        env_icon = "⛔️"
+        if ["en_US.UTF-8", "de_DE.UTF-8"].include?(ENV[e]) || !ENV[e]
```

Comment:
How about we just test that it is ending in `.UTF-8` instead? I've read reports from users using it with all kinds of UTF-8 locales and it fixed their issues.